com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -497,7 +478,7 @@ void DrawStackSettings()
             if (m_StackSettingsFoldout.value)
             {
                 m_LayerList.DoLayoutList();
```

Comment:
Got exceptions in console from line 480.

Steps:
1. new URP template (change manifest.json)
2. set layout to default
3. right click "inspector" > add tab > inspector
4. select a camera
5. in one inspector, change camera Render Type = overlay
6. change camera Render Type = base

Rendering the Stack list on second inspector causes exceptions in console.